rhea/cores/fifo/fifo_fast.py

```diff
@@ -65,7 +65,7 @@ def fifo_fast(clock, reset, fbus, use_srl_prim=False):
         # the FIFO, only a read address is accounted.
         @always(clock.posedge)
         def rtl_srl_in():
-            if srlce:
+            if srlce and (not fbus.full):
```

Comment:
Sure, I'll remove this one after you tell me if you want the one below this removed too.